ui/lib/core/addon/components/download-csv.js

```diff
@@ -0,0 +1,17 @@
+import Component from '@glimmer/component';
+import layout from '../templates/components/download-csv';
+import { setComponentTemplate } from '@ember/component';
+import { action } from '@ember/object';
+
+class DownloadCsvComponent extends Component {
```

Comment:
nice! Could you add documentation to this new component?

Reply:
Done.